app/views/users/show.html.erb

```diff
@@ -1,5 +1,20 @@
 <section class="user-section">
   <h2><%= "Name: #{@user.name}" %></h2>
+
+  <% if current_user != User.find(@user.id)%>
+
+    <% if show_add_friend(@user) %>
+      <%= link_to 'Invite to friendship', {:controller => 'friendships', :action => 'create', :id => @user.id}, method: :post, class: 'profile-link' %>
+    <% end %>
+
+    <% if show_already_friend(@user) %>
+      <span> | you are friends</span>
+    <% end %>
```

Comment:
Extract the logic in Users view into helper methods.

Reply:
Issue solved